src/runtime/test/manifest-test.ts

```diff
@@ -1982,15 +1982,19 @@ resource SomeName
       assert.lengthOf(manifest.particles, 1);
       const particle = manifest.particles[0];
       assert.isEmpty(particle.trustChecks);
-      assert.equal(particle.trustClaims.size, 2);
+      assert.equal(particle.trustClaims.length, 2);
       
-      const claim1 = particle.trustClaims.get('output1');
-      assert.equal(claim1.handle.name, 'output1');
-      assert.equal((claim1.expression as ClaimIsTag).tag, 'property1');
+      const claim1 = particle.trustClaims.find(claim => {
+        return (claim.handle.name === 'output1');
+        });
+      assert(claim1);
+      assert.equal((claim1.claims[0] as ClaimIsTag).tag, 'property1');
 
-      const claim2 = particle.trustClaims.get('output2');
-      assert.equal(claim2.handle.name, 'output2');
-      assert.equal((claim2.expression as ClaimIsTag).tag, 'property2');
+      const claim2 = particle.trustClaims.find(claim => {
+        return (claim.handle.name === 'output2');
+        });
+      assert(claim2);
```

Comment:
Same here, one line lambda, and isNotNull. Here and throughout file

Reply:
Done. Thanks.